partner_daily_notes/models/res_partner.py

```diff
@@ -22,6 +22,9 @@
 from odoo import models, fields, api, _
 from datetime import datetime, timedelta
 import logging
+import pytz
+
+
```

Comment:
PEP 8: E303 too many blank lines (3)
There should only be one empty lines here.